charts/gateway/templates/configmap.yaml

```diff
@@ -93,20 +93,22 @@ data:
   system-properties: |-
 {{ .Values.config.systemProperties | indent 4 }}
 {{- end }}
-{{- if .Values.config.additionalSystemProperties }}
-{{- range .Values.config.additionalSystemProperties }}
-    {{ .name }}={{ .value }}
-{{- end }}
-{{- end }}
 {{ if (eq (include "embedded.gemfire.enabled" .) "true") }}
     com.l7tech.external.assertions.embeddedgemfireprovider.locators={{- template "embedded.gemfire.locators" . }}
     com.l7tech.external.assertions.embeddedgemfireprovider.start.locator=false
+{{- if eq .Values.config.gemfire.embedded.auth.enabled false }}
+    com.l7tech.external.assertions.embeddedgemfireprovider.requireAuth=false
+{{- end }}
     # com.l7tech.external.assertions.embeddedgemfireprovider.locator.port=7011
     # com.l7tech.external.assertions.embeddedgemfireprovider.jmx.manager.port=7012
     # com.l7tech.external.assertions.embeddedgemfireprovider.cache.server.port=7013
     # com.l7tech.external.assertions.embeddedgemfireprovider.membership.start.port=7014
     # com.l7tech.external.assertions.embeddedgemfireprovider.membership.end.port=7016
-
+{{- end }}
+{{- if .Values.config.additionalSystemProperties }}
```

Comment:
Do we need this additionalSystemProperties and can't it just go into SystemProperties?

Reply:
This is existing implementation that append to system.properties